.travis.yml

```diff
@@ -5,8 +5,8 @@ go_import_path: gonum.org/v1/plot
 
 # Versions of go that are explicitly supported by gonum.
 go:
+ - 1.16.x
```

Comment:
We can't have nice things on Windows.

Reply:
indeed...

Reply:
Not clear to me why I put this comment here, but glad you figured out what I meant.